pymc/backends/report.py

```diff
@@ -13,6 +13,7 @@
 #   limitations under the License.
 
 import dataclasses
+import itertools as it
```

Comment:
just import chain, no need for the it alias?

Reply:
I will do it if you have a strong opinion about it, but I am a bit against extracting functions from their namespaces in general, I think it makes it more difficult to understand where they come from. Especially in the context of PyMC where "chain" can have a whole different meaning…

Reply:
I think the bigger objection here is the alias. How about just `import itertools`?

I think the similarity is giving @ricardoV94 flashbacks to `at` :joy:

Reply:
Yes my problem is the alias

Reply:
I thought that was a standard alias. I'll remove it.